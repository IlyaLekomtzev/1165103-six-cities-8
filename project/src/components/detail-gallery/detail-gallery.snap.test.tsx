import { render } from '@testing-library/react';
import DetailGallery from './detail-gallery';

const images = [
  'https://images.unsplash.com/photo-1636192677130-83a4cbd0f7d3?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1470&q=80',
  'https://images.unsplash.com/photo-1627110563676-7f534cbab296?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1074&q=80',
  'https://images.unsplash.com/photo-1635886534663-3d258f58d586?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1470&q=80',
];

describe('Component: DetailGallery', () => {
  it('should render correctly', () => {
    const {container} = render(<DetailGallery images={images} />);
    expect(container).toMatchSnapshot();
  });
});
