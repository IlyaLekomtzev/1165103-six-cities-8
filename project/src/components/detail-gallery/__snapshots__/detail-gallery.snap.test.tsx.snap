// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Mistakes should render correctly 1`] = `
<div>
  <div
    class="property__gallery"
  >
    <div
      class="property__image-wrapper"
    >
      <img
        alt="Offer studio"
        class="property__image"
        src="https://images.unsplash.com/photo-1636192677130-83a4cbd0f7d3?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1470&q=80"
      />
    </div>
    <div
      class="property__image-wrapper"
    >
      <img
        alt="Offer studio"
        class="property__image"
        src="https://images.unsplash.com/photo-1627110563676-7f534cbab296?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1074&q=80"
      />
    </div>
    <div
      class="property__image-wrapper"
    >
      <img
        alt="Offer studio"
        class="property__image"
        src="https://images.unsplash.com/photo-1635886534663-3d258f58d586?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1470&q=80"
      />
    </div>
  </div>
</div>
`;
